types/react-photoswipe/index.d.ts

```diff
@@ -0,0 +1,223 @@
+// Type definitions for react-photoswipe 1.3
+// Project: https://github.com/minhtranite/react-photoswipe
+// Definitions by: kwzm <https://github.com/kwzm>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import * as React from 'react';
+import { Item, Options } from 'photoswipe';
+
+export function onClose(): void;
+
+export function beforeChange(instance: PhotoSwipe): void;
+
+export function afterChange(instance: PhotoSwipe): void;
+
+export function imageLoadComplete(instance: PhotoSwipe, index: number, item: Item): void;
```

Comment:
These functions don't seem to be exported top level in the module. It looks like it would be more appropriate to just define these types without exporting them.

Reply:
Ok, I was just in case.